### PR TITLE
Minor bug fixes for DG

### PR DESCRIPTION
**Pull request by yucheng11122017**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ Step 3. CONNECTIONS displays a list of available commands in `ResultDisplay`.
 
 Step 4. The user decides to view the usage of `add` to learn to add a contact, and executes `help add`.
 
-Step 5. CONNECTIONS will display a detailed help message on the usage of the `add` command in `ResultDisplay`.
+Step 5. CONNECTIONS will displays a detailed help message on the usage of the `add` command in `ResultDisplay`.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:389]: display

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -455,7 +455,7 @@ Step 10. The headers and rows are written to the CSV file that is specified by t
 Allows the user to recover partial data if the data file becomes corrupted. 
 
 #### Proposed Implementation
-If the data file is corrupted, CONNECTIONS will use an empty data file upon start up. 
+Currently if the data file is corrupted, CONNECTIONS will use an empty data file upon start up.
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:458]: Currently, if the ..... or remove currently?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1194,29 +1194,34 @@ testers are expected to do more *exploratory* testing.
...
-    1. Test case: `tag 2 t/<existing tag in contact at index 2>`<br>
+    1. Test case: `tag 2 t/friends`<br>
        Expected: No changes to the contact. Warning details are shown in the status message.
        
-    1. Test case: `tag 2 t/<tag not in contact at index 2 > t/<existing tag in contact at index 2>`<br>
+    1. Test case: `tag 2 t/party t/friends`<br>
         Expected: New tag is added to the contact. Warning details are shown in the status message.
 
+<<<<<<< HEAD
+    1. Other incorrect tag commands to try: `tag`, `tag -1 t/<valid tag name>`, `tag 1 t/<tag name longer than 60 characters>`, `...`.<br>
```
Comment (cookiedan42) [inline docs/DeveloperGuide.md:1209]: merge conflict committed

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1194,29 +1194,34 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: No changes to contacts. Error details in status message.
+=======
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1211]: merge conflict?

**Review by cookiedan42 — APPROVED** [review]
Fix the committed merge conflict and can merge

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1194,29 +1194,34 @@ testers are expected to do more *exploratory* testing.
...
     1. Other incorrect tag commands to try: `tag`, `tag -1 t/<valid tag name>`, `tag <tag name longer than 60 characters>`, `...`.<br>
        Expected: No changes to contacts. Error details are shown in the status message.
+>>>>>>> 6e2c614ac8ae81b9023570e64dc53d7ca80d2655
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1214]: whats this ah

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -1194,29 +1194,34 @@ testers are expected to do more *exploratory* testing.
...
 ### Untagging a contact
 1. Untagging a contact
 
     1. Prerequisites: Contact at index 1 has tags johnBirthday and party and contact at index 2 has no tags.
 
     1. Test case: `untag 1 t/johnBirthday t/party`<br>
-       Expected: Remove specified tags from contact at index 1. Details of the removed tags are shown in the status message.
+       Expected: Removes specified tags from contact at index 1. Details of the removed tags are shown in the status message.
 
-    1. Test case: `untag 2 t/<non-existent tag in contact at index 2>`<br>
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1219]: think the prev one is ok

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1194,29 +1194,34 @@ testers are expected to do more *exploratory* testing.
 
 1. Tagging a contact with multiple tags
 
-    1. Prerequisites: Contact at index 1 has no tags and contact at index 2 has no tags.
+    1. Prerequisites: Contact at index 1 has no tags and contact at index 2 has tag of friends.
 
     1. Test case: `tag 1 t/johnBirthday t/party`<br>
        Expected: Tags added to contact at index 1. Details of the tags added to contact are shown in the status message.
 
-    1. Test case: `tag 2 t/<existing tag in contact at index 2>`<br>
```
Comment (leezhixuan) [inline docs/DeveloperGuide.md:1202]: think the prev one is better

**Review by leezhixuan — APPROVED** [review]
LGTM <3

**Pull request merged**